media-video/obs-studio/obs-studio-9999.ebuild

```diff
@@ -55,6 +63,22 @@ DEPEND="
 	x11-libs/libXrandr
 	x11-libs/libxcb
 	alsa? ( media-libs/alsa-lib )
+	browser? (
+		app-accessibility/at-spi2-atk
+		dev-libs/atk
+		dev-libs/expat
+		dev-libs/glib
+		dev-libs/nspr
+		dev-libs/nss
+		media-libs/fontconfig
+		x11-libs/libXScrnSaver
+		x11-libs/libXcursor
```

Comment:
Possible nitpick: it looks like some like to sort upper case before lower case, while some disregard case.

I might prefer disregarding case, but I'm not even entirely sure at this time.  In any... case... it will ultimately be up to the person who'll do the actual commit (which I'm unable to, still).